### PR TITLE
new ebuild ABI_connectivity_data

### PR DESCRIPTION
**Pull request by setina42**
New ebuild for packaged connectivity data from the Allen Mouse Brain Institute
Package-Manager: Portage-2.3.54, Repoman-2.3.12

### CODE REVIEW

**Comment by TheChymera** [comment]
thank you for your contribution! To ensure a basic quality check for your ebuilds and commit message, please commit this change not directly via git but by navigating to the ebuild dir and [using repoman](https://wiki.gentoo.org/wiki/Gentoo_GitHub#Usage_of_repoman).

File: sci-biology/ABI_connectivity_data/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	<maintainer type="person">
+		<email>tina.seg@bluewin.ch</email>
+		<name>Tina Segessemann</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>sci@gentoo.org</email>
+		<name>Gentoo Science Project</name>
+	</maintainer>
+	<longdescription lang="en">
+		Connectivity data from the Allen Mouse Brain data portal.
```
Comment (TheChymera) [inline sci-biology/ABI_connectivity_data/metadata.xml]: Plase make the longdescription longer.

File: sci-biology/ABI_connectivity_data/ABI_connectivity_data-0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+DESCRIPTION="Collection of connectivity data form the Allen Mouse Brain data portal"
+HOMEPAGE="https://github.com/IBT-FMI/ABI_connectivity_data"
+SRC_URI="
+	http://chymera.eu/distfiles/${P}.tar.xz
```
Comment (TheChymera) [inline sci-biology/ABI_connectivity_data/ABI_connectivity_data-0.1.ebuild]: one line is enough at this stage, break this up accross lines only when you need to.

File: sci-biology/ABI_connectivity_data/ABI_connectivity_data-0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+DESCRIPTION="Collection of connectivity data form the Allen Mouse Brain data portal"
```
Comment (TheChymera) [inline sci-biology/ABI_connectivity_data/ABI_connectivity_data-0.1.ebuild]: merely a style comment, but I think “Collection” here is unnecessary. “Connectivity data form the Allen Mouse Brain portal” provides the same level of information and is more concise.

**Comment by TheChymera** [comment]
@setina42 please delete the duplicated files with the old naming scheme

**Comment by TheChymera** [comment]
https://gitweb.gentoo.org/proj/sci.git/commit/?id=b1d0216bbbebaaa7a256320c7ec4056c4fb3c855

**Comment by TheChymera** [comment]
@setina42 please, in the future, remember to add a comment regarding what you did to the package on the commit line at the start of which repoman prints the package name, as you can see [here](https://gitweb.gentoo.org/proj/sci.git/log/).

**Comment by heroxbd** [comment]
@TheChymera Can we close this PR now?

**Comment by heroxbd** [comment]
Looks like so.

**Pull request closed without merging**